scalatest/src/main/scala/org/scalatest/PrivateMethodTester.scala

```diff
@@ -166,7 +166,7 @@ trait PrivateMethodTester {
 
         val isInstanceMethod = !Modifier.isStatic(m.getModifiers())
         val simpleName = m.getName
-        val paramTypes = m.getParameterTypes
+        val paramTypes: Seq[Class[_]] = m.getParameterTypes.map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.